Add documentation for void confirmation endpoint (#475)
centrapay/centrapay-docs: src/api/payment-requests/payment-requests.md

| ----------------- | ------------------------------------------------------------------------------------------------------------- |
| request           | [Payment Request][] was created.                                                                              |
| payment           | [Payment Request][] was paid.                                                                                 |
| refund            | Funds were returned to the shopper.                                                                           |
| cancellation      | [Payment Request][] was cancelled by the merchant or the shopper.                                             |
| expiry            | [Payment Request][] wasn't paid before time out.                                                              |
| accept-condition  | A [Payment Condition][] was accepted.                                                                         |
| decline-condition | A [Payment Condition][] was declined.                                                                         |

> | void-confirmation | A Confirmation Activity was cancelled by the merchant and any transferred funds were returned to the shopper. |

Reviewer: AFAIK funds are not returned to the shopper. This is under active discussion, but here's a go at some other words:

```suggestion
| void-confirmation | A Confirmation was reversed by the merchant. |
```

| 403    | {% break _ INVALID_AMOUNT   %}             | The confirmation is greater then the remaining funds on the authroization                                                                                           |
| 403    | {% break _ IDEMPOTENT_OPERATION_FAILED %}  | There has already been a confirmation against the Payment Request with the same idempotencyKey but different content. We recommend that you void this confirmation. |

<a name="void-confirmation"></a>
### Void a confirmation against a Pre Auth Payment Request **EXPERIMENTAL**

> Voiding a confirmation will reverse the confirmation. An `idempotencyKey` is required as a reference to the confirmation to be voided.

Reviewer: We should document a 200 here even when we don't know what the idempotency key means. So if a void-confirm comes through, we'd say something like "We've done everything we can with this in our system to back out any confirmation associated with that idempotency key".
